style.css

```diff
@@ -363,8 +363,18 @@ label[for="nav_toggle"] { /* The MENU button */
 .fn {
     transition: background-color 100ms ease-in-out;
 }
-.fn:hover {
-    background-color: hsla(calc(75 - var(--fn) * 10), 80%, calc(100% - 3%*var(--fn)));
+
+.def .fn:hover {
+    background-color: hsla(calc(187 - var(--fn) * 10), 61%, calc(100% - 3%*var(--fn)));
```

Comment:
Should `--fn` start as `0` instead of `1`, so that the color matches exactly at the top level?

Reply:
Yes, I think it's a slight improvement.

Reply:
Can you update the screenshot?